Dockerfile-client

```diff
@@ -0,0 +1,27 @@
+FROM node:16 as build
+
+MAINTAINER SDF Ops Team <ops@stellar.org>
+
+WORKDIR /app
+
+ARG REACT_APP_AMPLITUDE_KEY
+ENV REACT_APP_AMPLITUDE_KEY $REACT_APP_AMPLITUDE_KEY
+
+ARG REACT_APP_SENTRY_KEY
+ENV REACT_APP_SENTRY_KEY $REACT_APP_SENTRY_KEY
```

Comment:
could you actually rm the references to `REACT_APP_SENTRY_KEY` and `REACT_APP_AMPLITUDE_KEY`? These are no longer used